shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/data/repository/FeedingRepositoryImpl.kt

```diff
@@ -31,6 +33,7 @@ internal class FeedingRepositoryImpl(
     private val usersRepository: UsersRepository
 ) : FeedingRepository {
 
+    override fun getFeedStateFlow() = MutableSharedFlow<DomainFeedState>()
```

Comment:
Don't expose mutable objects to other classes, this method must return immutable instance of SharedFlow. To update it create a separate method and usecase.